src/serac/numerics/functional/functional_qoi.inl

```diff
@@ -453,7 +453,7 @@ private:
         for (axom::IndexType e = 0; e < K_elem.shape()[0]; e++) {
           for (axom::IndexType j = 0; j < K_elem.shape()[2]; j++) {
             auto [index, sign] = LUT(e, j);
-            gradient_L_[index] += sign * K_elem(e, 0, j);
+            gradient_L_.GetData()[index] += sign * K_elem(e, 0, j);
```

Comment:
wait, what-- `mfem::Vector` doesn't define `operator[]` (?!)

Reply:
This one was less "easy" than the others and I might have gotten the wrong answer.  Let me reproduce the warning/error I was getting.

Reply:
I wasn't complaining, just surprised to realize that the `operator[]` must have been coming from the implicit conversion to `double *`.

Reply:
I guess they have `mfem::Vector::operator()(int i)`, does that work?

Reply:
I believe the answer is yes, so we should be using that.

Reply:
Pushed a better fix.